docs/commands/functions.md

```diff
@@ -22,4 +30,9 @@ gef➤  deref $_heap(0x20) l4
 0x0000000000602028│+0x08: 0x0000000000020fe1
 0x0000000000602030│+0x10: 0x0000000000000000
 0x0000000000602038│+0x18: 0x0000000000000000
+gef➤  deref $_base(\"libc\") l4
```

Comment:
Do we really have to escape quotes?

Reply:
yes, unfortunately that is due to sh/bash. We could circumvent it by adding an additional parsing layer (but it would add lines of code and add the possibility of introducing bugs if not done carefully). But I can try if you think the usability is worth it?

Reply:
thanks for playing with it.